File: C /8.c
#include <stdio.h>

int main()
{
    int i, j, p, k;

    for (i = 0; i < 5; i++)
    {
        for (j = i; j < 5; j++)
        {
            printf(" ");
        }
        for (k = 0; k <= i; k++)
        {
            printf("*");
        }
        for (p = 0; p < i; p++)
        {
            printf("*");
        }

        printf("\n");
    }
    for (i = 5; i >= 0; i--)
    {
        for (j = i; j < 5; j++)
        {
            printf(" ");
        }
        for (k = 0; k <= i; k++)
        {
            printf("*");
        }
        for (p = 0; p < i; p++)
        {
            printf("*");
        }

        printf("\n");
    }

    return 0;
}

    // OUTPUT:-
    //       *    
    //      ***
    //     *****
    //    *******
    //   *********
    //  ***********
    //   *********
    //    *******
    //     *****
    //      ***
    //       *

File: Python /8.py
for i in range(5, -1, -1):
    for j in range(i, 5):
        print(" ", end="")        
    for p in range(i + 1):
        print("*", end="")   
    for k in range(i):
        print("*", end="")        
    print()

    # OUTPUT:-
    # ***********
    #  *********
    #   *******
    #    *****
    #     ***
    #      *
